README.rst

```diff
@@ -252,6 +252,46 @@ for the `RenkuLab <https://renkulab.io>`_ web UI. The service can be deployed in
 production as a Helm chart (see `helm-chart <./helm-chart/README.rst>`_.
 
 
+Deploying locally
+-----------------
+
+To test the service functionality you can deploy it quickly and easily using
```

Comment:
can you install docker-compose with setup.py??

Reply:
it's just a part of docker I thought

Reply:
I added a line at the top that mentions needing docker for service development - not sure if that's what you had in mind?

Reply:
https://pypi.org/project/docker-compose/

Reply:
but we don't actually use that, do we?

Reply:
oh wait, I thought that was a python wrapper for docker compose, but it's actually written in python 🤦  - still since it's now just distributed with docker, we probably don't need to add it specifically to the requirements?

Reply:
I added it to setup.py under service, but it shouldn't really be there because it's not required to run the service. It also doesn't belong under `tests`... maybe we could have a `develop` set of requirements (basically `tests` but include packages like `docker-compose` and `ptvsd` that are not really needed for tests)? Could just rename `tests` I guess...